client/src/components/ReviewItem.jsx

```diff
@@ -42,7 +41,7 @@ class ReviewItem extends React.Component {
   }
 
   goToUserPage() {
-    alert("This button doesn't actually do anything because I didn't make a user page because that wasn't part of my role!");
+    alert("This button is not functional yet, and will be implemented in a future update");
```

Comment:
This seems more professional in the long run instead of saying its not your responsibility and instead just say its not implemented at the moment